### PR TITLE
Support for Vue

### PR DESCRIPTION
**Pull request by pax-k**
Implements
- packages/jazz-vue
- examples/chat-vue

### CODE REVIEW

**Review by aeplay — CHANGES_REQUESTED** [review]
Looks good to me, please may you add a changeset so it triggers a release PR?

File: packages/jazz-vue/src/createJazzVueApp.ts [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+        setup(props, { slots }) {
+            const ctx = ref<
+                BrowserContext<Acc> | BrowserGuestContext | undefined
+            >(undefined);
+
+            provide(JazzContextSymbol, ctx);
+
+            onMounted(async () => {
+                try {
+                    console.log("props.auth", props.auth);
```
Comment (gdorsi) [inline packages/jazz-vue/src/createJazzVueApp.ts]: **question:** do we want to keep this log?

File: packages/jazz-vue/src/createJazzVueApp.ts [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+                                  peer: props.peer,
+                                  storage: props.storage,
+                              },
+                    );
+
+                    ctx.value = {
+                        ...context,
+                        logOut: () => {
+                            context.logOut();
+                            ctx.value = undefined;
```
Comment (gdorsi) [inline packages/jazz-vue/src/createJazzVueApp.ts]: **observation:** Not sure if we need to create a new context on logout

@aeplay How do we handle the login -> logout -> login flow?

**Review by gdorsi — APPROVED** [review]
Awesome work!

Just left one question regarding the logout and context management.

File: packages/jazz-vue/src/createJazzVueApp.ts (hunk 2 above)
Comment (pax-k) [reply packages/jazz-vue/src/createJazzVueApp.ts]: Fixed

**Review by aeplay — APPROVED** [review]

**Pull request merged**